config/routes.rb

```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  # RESTful routes:
+  root to:'homepages#index'
```

Comment:
I don't think you can use "to" for the root; I think it's just:

```suggestion
  root 'homepages#index'
```